mana_mushlemet.py

```diff
@@ -0,0 +1,34 @@
+
+
+def is_complete(num_pieces: int) -> bool:
+    """
+    The function get a number and check if it complete meal
```

Comment:
Check your grammer :) 
The function gets a number and checks if it is a complete meal.